bigchaindb/common/schema/transaction_v2.0.yaml

```diff
@@ -63,7 +63,7 @@ definitions:
     - CREATE
     - TRANSFER
     - VALIDATOR_ELECTION
```

Comment:
We also need a generic name instead of `VALIDATOR_ELECTION` now.

Reply:
What purpose would that serve since the name would always be overriden by the specific election type?

Reply:
True, there is no purpose in such an operation..

But extending the base schema every time a new election is introduced does not sound necessary either.

I would not require the operation in the base spec. Every specific spec can require the particular operation itself.

Does not have to be done in this PR though.

Reply:
I don't think we do...

Since `Election` is a base class, and will be extended with distinct election classes, shouldn't each of those get their own `operations`? The `ValidatorElection` is still using `VALIDATOR_ELECTION` as it's operation.

Reply:
Yes. An extra point (we can address it later) is that `bigchaindb/common/schema/transaction_v2.0.yaml` does not have to require the operation to be from the given set. It just makes us extend this list every time a new operation is introduced.